internal/commands/config/list_profiles_internal.go

```diff
@@ -1,21 +1,44 @@
 package config_internal
 
 import (
+	"slices"
+
+	"github.com/fatih/color"
+	"github.com/pingidentity/pingcli/internal/logger"
 	"github.com/pingidentity/pingcli/internal/output"
 	"github.com/pingidentity/pingcli/internal/profiles"
 )
 
 func RunInternalConfigListProfiles() {
+	l := logger.Get()
+
 	profileNames := profiles.GetMainConfig().ProfileNames()
 	activeProfile := profiles.GetMainConfig().ActiveProfile().Name()
 
 	listStr := "Profiles:\n"
+
+	slices.Sort(profileNames)
+
+	output.SetColorize()
```

Comment:
This will happen in `output.Print()` anyways. Safe to remove?

Reply:
I tried this - the issue is that the colourisation has to be set first before the `activeFmt` use on line 28.  If colourisation happens after line 28 is run, the "no colour" is ignore